pkg/fetcher/fetcher.go

```diff
@@ -100,6 +100,21 @@ type URLFetcher struct {
 	options Options
 }
 
+// ErrorResponseBody struct
```

Comment:
Good point - then we can call it `RegistryErrorRespBody` or leave it as-is. Your call!